cmd/cored/default.go

```diff
@@ -0,0 +1,43 @@
+//+build !protected_db
+
+// This configuration includes access to the reset endpoint.
+package main
+
+import (
+	"chain/core/config"
+	"chain/core/coreunsafe"
+	"chain/database/pg"
+	"chain/env"
+	"chain/log"
+	"context"
+	"fmt"
+	"os"
+)
+
+var (
+	reset = env.String("RESET", "")
+)
+
+func init() {
+	config.BuildConfig.ProtectedDB = false
```

Comment:
I think the default values should be initialized here, or in config.go, but not both. If here, then there should be a comment in config.go saying "initialized in $CHAIN/cmd/cored/default.go".